python/lsst/meas/algorithms/matcherSourceSelector.py

```diff
@@ -132,46 +144,16 @@ def _isUsable(self, sourceCat):
             & self._goodSN(sourceCat) \
             & ~sourceCat.get(self.fluxFlagKey)
 
-
-@pexConfig.registerConfigurable("matcherPessimistic", sourceSelectorRegistry)
-class MatcherPessimisticSourceSelectorTask(MatcherSourceSelectorTask):
-    """Select sources that are useful for matching.
-
-    Good matching sources have high signal/noise, are non-blended. They need not
-    be PSF sources, just have reliable centroids. This inherited class adds
-    the removal of saturated, interpolated, and edge_key objects to the set of
-    bad flags. It is a temporary addition designed preserve the source selction
-    used in matchOptimisticB. Once matchPessimisticB is adopted as the default
-    source selector the class will be removed and the saturated, interpoalted, and
-    edge_key flags will be added to the matcherSourceSelector class.
-
-    TODO: Once DM-10399 is complete an RFC will be filed to make matchPessimisticB
-    the default matcher this class will replace matcherSourceSelector with this source
-    selector resulting in only one matcherSourceSeletor. The ticket describing
-    this work is DM-10800.
-    """
-    def _getSchemaKeys(self, schema):
-        """Extract and save the necessary keys from schema with asKey."""
-        MatcherSourceSelectorTask._getSchemaKeys(self, schema)
-
-        self.edgeKey = schema["base_PixelFlags_flag_edge"].asKey()
-        self.interpolatedCenterKey = schema["base_PixelFlags_flag_interpolatedCenter"].asKey()
-        self.saturatedKey = schema["base_PixelFlags_flag_saturated"].asKey()
-
-    def _isUsable(self, sourceCat):
+    def _isGood(self, sourceCat):
         """
         Return True for each source that is usable for matching, even if it may
         have a poor centroid.
 
         For a source to be usable it must:
-        - have a valid centroid
-        - not be deblended
-        - have a valid instFlux (of the type specified in this object's constructor)
-        - have adequate signal-to-noise
+        - Not be on a CCD edge.
+        - Not have an interpolated pixel value
```

Comment:
I believe this is only checking if the center is interpolated, not if the object has any interpolated pixels.